### PR TITLE
Fix housing model

### PR DESCRIPTION
**Pull request by joostveenema**
This model had a diverging loss due to a minus sign in the update step. This PR does also replace the custom meansquareerror function with `Flux.Losses.mse`.

### CODE REVIEW

File: other/housing/housing.jl [hunk 1]
```diff
@@ -67,24 +65,24 @@ function train(; kws...)
...
 
     for i = 1:500
       g = gradient(() -> loss(x_train, y_train), θ)
       for x in θ
-        update!(x, -g[x]*η)
+            update!(x, g[x]*η)
       end
       if i%100==0
           @show loss(x_train, y_train)
         end
```
Comment (CarloLucibello) [inline other/housing/housing.jl:81]: indentation issues

**Comment by CarloLucibello** [comment]
This example seems to be quite low level, maybe we should leave the custom mse there?

**Comment by joostveenema** [comment]
It is. But I think it makes more sense to show using a custom loss function in of the tutorials were it can not simply be replaced by one from the available functions in Flux. Removed it from this PR, but left the in the list of issues for now.

**Comment by DhairyaLGandhi** [comment]
If we add some comments that bring attention to the custom mse, that makes the same point, but tutorials with more complex loss functions are welcome of course

**Pull request merged**